### PR TITLE
HTML: Compute kind of widget

### PR DESCRIPTION
**Pull request by howard-e**
References whatwg/html#7004

### CODE REVIEW

**Comment by zcorpan** [comment]
`infrastructure/ tests` ought to be unrelated to this PR

**Comment by zcorpan** [comment]
@andruud can you review?

File: css/css-ui/compute-kind-widget-no-fallback-props-001.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  'border-top-right-radius',
+  'border-bottom-right-radius',
+  'border-bottom-left-radius',
+  'border-start-start-radius',
+  'border-start-end-radius',
+  'border-end-start-radius',
+  'border-end-end-radius',
+];
+
+const declarations = getComputedStyle(document.documentElement);
```
Comment (josepharhar) [inline css/css-ui/compute-kind-widget-no-fallback-props-001.html:88]: Is the idea here that we are making sure that all possible CSS properties don't affect the widget type, except the ones that we have in our explicit list?

**Comment by josepharhar** [comment]
Is merging this blocked on the csswg and whatwg/html spec PRs?

Also, would it make sense to make the filenames have "appearance" in their filename? I think it would make them easier to find and query for when looking for appearance related WPTs, since it looks like there is no dedicated directory for appearance tests in the css-ui directory.

File: css/css-ui/compute-kind-widget-no-fallback-props-001.html (hunk 1 above)
Comment (zcorpan) [reply css/css-ui/compute-kind-widget-no-fallback-props-001.html:88]: Exactly! We can explain that in a comment if that's helpful.
```suggestion
// Make sure that any supported property that is not in the above list
// does not affect the widget type.
const declarations = getComputedStyle(document.documentElement);
```

**Comment by zcorpan** [comment]
> Is merging this blocked on the csswg and whatwg/html spec PRs?

I think it would be nice to merge all 3 at the same time when they're all reviewed.

> Also, would it make sense to make the filenames have "appearance" in their filename? I think it would make them easier to find and query for when looking for appearance related WPTs, since it looks like there is no dedicated directory for appearance tests in the css-ui directory.

Certainly. Will fix.

File: css/css-ui/compute-kind-widget-no-fallback-props-001.html (hunk 1 above)
Comment (josepharhar) [reply css/css-ui/compute-kind-widget-no-fallback-props-001.html]: A comment would be helpful

**Review by josepharhar — APPROVED** [review]

**Comment by zcorpan** [comment]
`update-built` check failed, that needs to be fixed.

**Comment by zcorpan** [comment]
`wpt-chrome-dev-results` check now fails with "The GPU process has crashed 6 time(s)" :/

https://community-tc.services.mozilla.com/tasks/HUL4rwR0SDWs2qrsYxXGLA/runs/0/logs/public/logs/live.log#L1027

@foolip any idea?

**Comment by josepharhar** [comment]
I have filed a webkit bug to implement these changes: https://bugs.webkit.org/show_bug.cgi?id=233430

**Comment by zcorpan** [comment]
I tried rebasing on the latest `master` to see if the error in CI would go away, but it did not. I don't know what's up...

```
The GPU process has crashed 6 time(s)
```
https://community-tc.services.mozilla.com/tasks/SfckzfzUSwmykRfqhTZGmA/runs/0/logs/public/logs/live.log

@web-platform-tests/admins can you help?

**Comment by foolip** [comment]
@zcorpan I've worked around the issue with https://github.com/web-platform-tests/wpt/pull/31715 now. Rebasing this should  fix the problem.

But there's really no need, since only the Chrome jobs were failing, I'd just admin merge this, rather than waiting for CI to run again.

**Pull request merged**